### PR TITLE
Support new model

### PR DESCRIPTION
**Pull request by pierrestock**

### CODE REVIEW

**Comment by pierrestock** [comment]
Solves issue https://github.com/stanford-futuredata/megablocks/issues/46

File: csrc/indices.h [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
-// to write 32 elements per block per iteration.
-// This is the largest we're every likely to use
-// so we keep the blocks small.
-const int kThreadsPerBlock = 32;
+// This accounts for a maximum of
+// ffn_hidden_size = kThreadsPerBlock * 128.
+// Modify accordingly if ffn_hidden_size changes.
+// Increasing it would reduce a bit compute efficiency 
+// (to be benchmarked). 
+const int kThreadsPerBlock = 112;
```
Comment (tgale96) [inline csrc/indices.h:21]: Can we make this 128 so its a multiple of the warp size (32)?

**Comment by tgale96** [comment]
Added one comment - this should work fine as a quick fix to unblock you. 

I've been meaning to port this to Triton and I can also clean up the grid dimensions for different problem sizes as part of that :)

**Comment by tgale96** [comment]
FYI: I generalized the fix in #47. Thanks for isolating the issue to this kernel!

**Pull request closed without merging**